bot/commands/edit.js

```diff
@@ -0,0 +1,33 @@
+// Edits the message and logs the old content in #bot-log
+const {Command} = require('yuuko');
+
+module.exports = new Command('edit', async (msg, args, context) => {
+	if (args.length < 2) {
```

Comment:
editing a message to have no content isn't valid, so this should probably check for a minimum of 3 args instead

```suggestion
	if (args.length < 3) {
```

Reply:
Nevermind, just saw that channel ID is optional, ignore me